rollup.config.js

```diff
@@ -0,0 +1,29 @@
+import typescript from '@rollup/plugin-typescript';
+import dts from "rollup-plugin-dts";
+
+export default [
+  {
+    input: './src/index.ts',
+    output: {
+        file: 'index.js',
+        format: 'cjs'
+    },
+    plugins: [typescript()]
```

Comment:
I noticed some oddities in the build files (check line 120 of index.js - it contains a `p;` statement which is of course not valid in JS). This is why the tests are failing for node 10. Apparently later versions can tolerate that 🤔 

In any case, we can specify the build version as ES6 for the common js build, and that takes care of it.

```javascript
plugins: [typescript({
  target: "ES6"
})]
```